js/languages/en-US.json

```diff
@@ -806,6 +806,7 @@
   },
```

Comment:
@rmisio @jjeffryes I tested the scenario above and it worked correctly for me.

To be clear, I did the following:

1) Applied a 50% off coupon
2) Attempted to apply an additional coupon for 51% off (it triggered an error)
3) I removed the initial 50% off coupon
4) Attempted to apply the 51% off coupon (it worked!)